src/main/scala/org/allenai/plugins/DeployPlugin.scala

```diff
@@ -145,6 +152,7 @@ object DeployPlugin extends AutoPlugin {
   lazy val deployTask = deploy := {
     // Dependencies
     gitRepoClean.value
+    stageAndCacheKey.value
```

Comment:
Why don't you just use the task in `VersionInjectorPlugin` directly here? I don't see why you have a new task for this . . .